roles/cinder-control/tasks/v7k-monitoring.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: create v7k sshkey directory
+  file: dest=/etc/cinder/v7k/ssh/monitoring state=directory owner=sensu
+        group=sensu mode=0755
+
+- name: drop a cinder private key
+  template: src=etc/v7k/ssh/monitoring/id_rsa
+            dest=/etc/cinder/v7k/ssh/monitoring/id_rsa owner=sensu
+            group=sensu mode=0600
+
+- name: v7k canister status check
+  sensu_check: name=canister-status plugin=check_v7k_canister.py
+               args='-v {{v7k.management_ip}} -u {{v7k.users.monitoring.name}} -i /etc/cinder/ssh/monitoring/id_rsa -p {{v7k.ssh_port}}' use_sudo=true
+  notify: restart sensu-client
+
+- name: v7k canister load metrics
+  sensu_check: name=canister-load plugin=metrics_v7k.py
+               args='-v {{v7k.management_ip}} -u {{v7k.users.monitoring.name}} -i /etc/cinder/ssh/monitoring/id_rsa  -p {{v7k.ssh_port}}' use_sudo=true
+  notify: restart sensu-client
+  
```

Comment:
Please add related plugins in roles/common/templates/monitoring/sensu-sudoers, or it will face error of "sudo: no tty present and no askpass program specified".